src/autopas/namespaces.h

```diff
@@ -19,12 +19,22 @@ namespace autopas {
  */
 namespace ArrayMath {}
 
+/**
+ * In this namespace some helper functions for floating point std::array's can be found.
```

Comment:
```suggestion
 * In this namespace some helper functions for std::array can be found.
```

No (any more) restricted to floating point! :) please also adapt ArrayMath description.